src/sentry/static/sentry/app/views/settings/incidentRules/ruleConditionsForm.tsx

```diff
@@ -54,6 +62,53 @@ class RuleConditionsForm extends React.PureComponent<Props, State> {
     this.fetchData();
   }
 
+  generateFieldOptions() {
```

Comment:
Should we have a module level function in the column editor modules that does this work?

Reply:
Yes, we should.